docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -85,143 +85,143 @@ Remove the existing class definition and add the following code, which has two c
+## Choose a learning algorithm
 
-Add the following code into the `Train` method following the data processing code added in the previous step:
+After adding the data to the pipeline and transforming it into the correct input format, we select a learning algorithm (**learner**). The learner trains the model. We chose a **regression** task for this problem, so we use a <xref:Microsoft.ML.Trainers.FastTree.FastTreeRegressionTrainer> learner, which is one of the regression learners provided by ML .NET.
 
-```csharp
-pipeline.Add(new FastTreeRegressor());
-```
+<xref:Microsoft.ML.Trainers.FastTree.FastTreeRegressionTrainer> learner utilizes gradient boosting. Gradient boosting is a machine learning technique for regression problems. It builds each regression tree in a step-wise fashion. It uses a pre-defined loss function to measure the error in each step and correct for it in the next. The result is a prediction model that is actually an ensemble of weaker prediction models. For more information about gradient boosting, see [Boosted Decision Tree Regression](/azure/machine-learning/studio-module-reference/boosted-decision-tree-regression).
 
-You added all the preceding steps to the pipeline as individual statements, but C# has a handy collection initialization syntax that makes it simpler to create and initialize the pipeline. Replace the code you added so far to the `Train` method with the following code:
+Add the following code into the `Train` method to add the `FastTreeRegressionTrainer` to the data processing code added in the previous step:
 
-[!code-csharp[CreatePipeline](../../../samples/machine-learning/tutorials/TaxiFarePrediction/Program.cs#3 "Create and initialize the learning pipeline")]
+[!code-csharp[FastTreeRegressionTrainer](../../../samples/machine-learning/tutorials/TaxiFarePrediction/Program.cs#10 "Add the FastTreeRegressionTrainer")]
 
 ## Train the model
 
-The final step is to train the model. Until this point, nothing in the pipeline has been executed. The `pipeline.Train<TInput, TOutput>` method produces the model that takes in an instance of the `TInput` type and outputs an instance of the `TOutput` type. Add the following code into the `Train` method:
+The final step is to train the model. We train the model, <xref:Microsoft.ML.Runtime.Data.TransformerChain>, based on the dataset that has been loaded and transformed. Once the estimator has been defined, we train the model using the <xref:Microsoft.ML.Runtime.Data.EstimatorChain`1.Fit> while providing the already loaded training data. This returns a model to use for predictions. `pipeline.Fit()` trains the pipeline and returns a `Transformer` based on the `DataView` passed in. The experiment is not executed until this happens.
```

Comment:
You need the `%601` instead of the back tick.  Several locations with build warnings in this PR.

Reply:
Yes, sorry, had not gotten to those yet.

Reply:
The rest should resolve with API release

Reply:
Clean build, @BillWagner!